aw_core/transforms.py

```diff
@@ -100,98 +99,71 @@ def filter_period_intersect(events, filterevents):
 
     return filtered_events
 
-def include_labels(events, labels):
+def include_keyvals(events, key, vals):
     filtered_events = []
     for event in events:
         match = False
-        for label in labels:
-            if label in event["label"]:
+        for val in vals:
+            if key in event.data and val == event.data[key]:
                 match = True
         if match:
             filtered_events.append(event)
     return filtered_events
 
-def exclude_labels(events, labels):
+def exclude_keyvals(events, key, vals):
```

Comment:
Better to just implement include/exclude difference as a keyword argument `exclude=False` and rename the funktion to `filter_keyvals` or something.